managed/services/telemetry/config.default.yml

```diff
@@ -225,6 +225,22 @@ telemetry:
       - metric_name: "mongodb_storage_engine_type"
         label: engine
 
+  - id: MongoDBIsEncryptionAtRestEnabled
+    source: VM
+    query: (group(mongodb_up) by (service_name)) * on(service_name) group_left(service_name) mongodb_ss_encryptionAtRest_encryptionEnabled OR (group(mongodb_up) by (service_name)) - 1
+    summary: "Returns 1 if encryption at rest is enabled on the mongo instance, otherwise returns 0"
+    data:
+      - metric_name: "mongodb_encryption_at_rest_enabled"
+        value: 1
+
+  - id: MongoDBIsEncryptionType
```

Comment:
I believe it is supposed to be `MongoDBEncryptionType`

Reply:
or  `MongoDBIsEncryptionTypeEnabled`

Reply:
Fixed that, thanks